crates/bevy_ecs/src/system/commands.rs

```diff
@@ -15,7 +19,9 @@ pub trait Command: Send + Sync + 'static {
 /// A queue of [`Command`]s.
 #[derive(Default)]
 pub struct CommandQueue {
-    commands: Vec<Box<dyn Command>>,
+    pub(crate) commands: Vec<Box<dyn Command>>,
+    #[allow(unused)]
+    pub(crate) system_name: Option<Cow<'static, str>>,
```

Comment:
```suggestion
    #[cfg(feature = "command_panic_origin")]
    pub(crate) system_name: Option<Cow<'static, str>>,
```

If I'm not using the `command_panic_origin` feature, I don't want to pay for the extra overhead of when it is turned on.

Reply:
I think the `Cow` should be very cheap here?

Reply:
Yes it may be cheap, but it's not 100% overhead free.
If we're already conditionally exposing the behavior behind a feature flag we may as well use it. 
Especially if it's only a minimal amount of code changes.